src/runtime/driver/mod.rs

```diff
@@ -122,6 +122,21 @@ impl Driver {
         ))
     }
 
+    pub(crate) fn submit_op_2(&mut self, sqe: squeue::Entry) -> usize {
+        let index = self.ops.insert();
+
+        // Configure the SQE
+        let sqe = sqe.user_data(index as _);
+
+        // Push the new operation
+        while unsafe { self.uring.submission().push(&sqe).is_err() } {
```

Comment:
Maybe say what unsafe contract is being upheld.

Reply:
This was copied from the earlier stuff. When I do a cleanup PR later when I've moved over all the ops, I can add this.